packages/playwright-core/src/server/socksClientCertificatesInterceptor.ts

```diff
@@ -274,6 +276,13 @@ export class ClientCertificatesProxy {
     loadDummyServerCertsIfNeeded();
   }
 
+  getProxyAgent(host: string, port: number) {
+    const proxyFromEnv = getProxyForUrl(`https://${host}:${port}`);
+    if (proxyFromEnv)
```

Comment:
Why can't the user do it in their configuration?

Reply:
Browsers make it work out of the box, would be great if we can make it work out of the box as well to match browser behaviour.